containers/ddev-xhgui/Dockerfile

```
@@ -1,20 +1,4 @@
-FROM xhgui/xhgui:0.21 AS xhgui-docker
-
-FROM scratch AS ddev-xhgui
-
-### --------------------------------xhgui-docker----------------------------------
-### XHGui Docker image creates a volume for /run/nginx, which we want to remove.
-### We copy all files from xhgui/xhgui, and set ENV, WORKDIR, CMD, EXPOSE.
-### Source https://github.com/perftools/xhgui/blob/0.21.x/Dockerfile
-COPY --from=xhgui-docker / /
-ENV PHP_INI_DIR=/etc/php7
-ARG APPDIR=/var/www/xhgui
-ARG WEBROOT=$APPDIR/webroot
-WORKDIR $APPDIR
-RUN mkdir -p /run/nginx
-CMD ["sh", "-c", "nginx && exec php-fpm"]
-EXPOSE 80
-### ------------------------------END xhgui-docker--------------------------------
+FROM xhgui/xhgui:0.23 AS ddev-xhgui
 
 RUN apk add bash curl
```

```suggestion
RUN apk add --no-cache bash curl
```

docker best practices not to store INDEX files to container.